### ISSUE TITLE
A little assistance

### ISSUE THREAD

**Issue by ebosetale214** [binary-com/binary-bot#1822]
Pls who can help with an option bot with high profit margin without martingale. Thanks alot

**Comment by hotmatrixx** [comment]
That's the dream, my friend. We all want that. 

To be honest with you, anyone who had a winning strat that they KNEW worked that well is probably not going to share it with you. At least, I won't. It's something that the individual has worked on. in my case, for a little over 3 years to find something that works consistently for me.

The most likely way you'll find help here, is to start creating your own strategies, and maybe asking us for help on how to code it.

My advice is don't ever buy other people's bots in here. Most people who sell bots on a forum like this, the bots probably don't work, or else they wouldn't need to sell them.

**Comment by ebosetale214** [comment]
Thanks@ hot matrix. I just want to know if that exist at all.talking  of strategy, is it on only martingale we can build a not. Is there no other way

**Comment by hotmatrixx** [comment]
it might be possible, but you need to keep some things in mind. a non-martingale bot would still need to recover any losses somehow, unless you could reliably predict your market more often than your payout.

you could up the stake upon a loss, with the intention of winning back over more than one trade - so if you lost one, you could divide that recovery up among 2 or more potential wins. that means that you'd lose less if you lost, but need more wins, also, to recover any potential losses.

Basically, you need a great strategy. failing that, you could try to focus on money management. remember that the winrates and payouts are geared against you. Just like a casino, the house always wins in the end.
